src/components/layout/SubscribedList.vue

```diff
@@ -60,10 +68,30 @@ export default {
     },
   },
   computed: mapGetters('jmap', ['mailboxes']),
+  mounted() {
+    const table = document.querySelector('table');
```

Comment:
it's just that the checkboxes feature are depreciated for this mdl version and that's the patch for making it working again.
See https://github.com/google/material-design-lite/wiki/Deprecations#automatic-selection-checkboxes for more informations about it..